cockatrice/src/tab_server.cpp

```diff
@@ -71,7 +71,11 @@ void RoomSelector::processListRoomsEvent(const Event_ListRooms &event)
     const int roomListSize = event.room_list_size();
     for (int i = 0; i < roomListSize; ++i) {
         const ServerInfo_Room &room = event.room_list(i);
-        
+        QString roomPermissionDisplay = QString::fromStdString(room.privilegelevel()).toLower();
```

Comment:
Yes, you can't get away with your typical if , followed by if, followed by if method you generally enjoy doing.  We want to avoid testing conditions based on combinations of permissions and privileges and avoid others when certain conditions are met.  I'll see about adding comments to make things more clear.